helix-term/tests/test/commands.rs

```diff
@@ -311,3 +311,24 @@ async fn test_undo_redo() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_extend_line() -> anyhow::Result<()> {
+    // extend with line selected then count
+    test((
+        platform_line("#[a|]#\na\na\n\n").as_str(),
+        "x2x",
+        platform_line("#[a\na\na\n|]#\n").as_str(),
+    ))
```

Comment:
I think the `indoc!` format for these would be a little more readable, for example https://github.com/helix-editor/helix/blob/63dcaae1b9083396fb3faaef9eaa2421f7e48fb9/helix-term/tests/test/commands.rs#L101-L115

The `\n` newlines make it hard to tell which lines are selected at a glance

Reply:
Fixed now